routes/reviews.js

```diff
@@ -96,4 +96,30 @@ router.post('/', (req, res) => {
     });
 });
 
+
+
+router.delete('/', (req, res) => {
+   
+
+    //The film's ID of the query is saved in ReviewId
+    var ReviewId=req.params.imbId;
```

Comment:
There is a typo here I think. It should be `req.params.imdbId`. Also, variable names should follow lowerCamelCase rule: `var reviewId`